python/paddle/fluid/contrib/slim/quantization/quant2_int8_mkldnn_pass.py

```diff
@@ -336,6 +338,9 @@ def _is_int8_weights(op_node, weight_name):
                 self._dequantize_op_weights(graph, op, "Filter", "Output")
             elif op.name() in self._mul_ops and _is_int8_weights(op, "Y"):
                 self._dequantize_op_weights(graph, op, "Y", "Out")
+            elif op.name() in self._matmul_ops and _is_int8_weights(op, "Y"):
```

Comment:
It looks like you can combine this part into one and just check `op.name() in [self._mul_ops, self._matmul_ops]`. What do you think?

Reply:
I can apply your idea, but I will concatenate those lists before for loop

Reply:
I wonder if we even need matmul_ops separate from mul_ops

Reply:
You're right, it looks like we are doing exactly the same things for them, so for me you can combine it.